bc_obps/reporting/schema/electricity_import_data.py

```diff
@@ -0,0 +1,47 @@
+from ninja import ModelSchema
+from pydantic import alias_generators
+from reporting.models import ElectricityImportData
+
+
+def to_snake(string: str) -> str:
+    return alias_generators.to_snake(string)
```

Comment:
this code is not useful, it just passes on the call. We can just do `from pydantic.alias_generators import to_snake` and use that in the schema below